### PR TITLE
Jvne theme!

### PR DESCRIPTION
**Pull request by Lylythii**
One more Prism theme, while I'm on a trip of re-styling apps!

### CODE REVIEW

File: themes/Jvne/theme.json.license [hunk 1]
```diff
@@ -0,0 +1,3 @@
+SPDX-FileCopyrightText: 2013-2021 MultiMC Contributors
+
+SPDX-License-Identifier: Apache-2.0
```
Comment (DioEgizio) [inline themes/Jvne/theme.json.license:3]: Is this the correct copyright/license notice?

File: themes/Jvne/themeStyle.css [hunk 2]
```diff
@@ -0,0 +1,7 @@
+/*
+ * SPDX-FileCopyrightText: 2013-2021 MultiMC Contributors
+ *
+ * SPDX-License-Identifier: Apache-2.0
```
Comment (DioEgizio) [inline themes/Jvne/themeStyle.css:4]: Same here

File: themes/Jvne/theme.json.license (hunk 1 above)
Comment (Scrumplex) [inline themes/Jvne/theme.json.license:3]: You used MIT in your README, so I would probably expect this to be MIT as well?

File: themes/Jvne/themeStyle.css (hunk 2 above)
Comment (Scrumplex) [inline themes/Jvne/themeStyle.css:4]: ditto

**Comment by Lylythii** [comment]
My bad! Didn't notice.

**Review by Scrumplex — APPROVED** [review]

**Comment by AshtakaOOf** [comment]
Why is this not being merged?

**Pull request merged**